js/controller.js

```diff
@@ -259,6 +269,19 @@ OO.plugin("Html5Skin", function (OO, _, $, W) {
 
       this.accessibilityControls = new AccessibilityControls(this); //keyboard support
       this.state.screenToShow = CONSTANTS.SCREEN.INITIAL_SCREEN;
+
+      if (this.getVrParams && this.getVrParams()) {
+        this.state.isVideo360 = true;
+        this.mb.subscribe(OO.EVENTS.DIRECTION_CHANGED, 'customerUi', _.bind(this.setViewingDirection, this));
+      }
+    },
+
+    getVrParams: function(){
+      var playerParam = this.state.playerParam;
+      var bitWrapper = playerParam ? playerParam['bit-wrapper'] : null;
```

Comment:
We should define a generic way to relay this information from the video plugin to the skin. We do not want to add dependencies on bitmovin in the skin files.